### PR TITLE
stonewall:0.1.0

### PR DESCRIPTION
**Pull request by coco33920**
I am submitting
- [X] a new package
- [ ] an update for a package

Description: Provides useful and accurate colour palettes for pride flags for use with gradients.

I have read and followed the submission guidelines and, in particular, I
- [X] selected a name in conformance with the guidelines
- [X] added a `typst.toml` file with all required keys
- [X] added a `README.md` with documentation for my package
- [X] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [X] ensured that my submission does not infringe upon the rights of a third party
- [X] tested my package locally on my system and it worked
- [X] named this PR as `name:version` of the submitted package
- [X] agree that my package will not be removed without good reason

### CODE REVIEW

File: packages/preview/pride/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "pride"
+version = "0.1.0"
+entrypoint = "pride.typ"
+authors = ["Charlotte Thomas"]
+license = "GPLv3"
+description = "Pride provides beautiful pride flag colours for gradients."
+repository = "https://github.com/coco33920/pride"
+exclude = ["example/*,"flags.png"]
```
Comment (istudyatuni) [inline packages/preview/pride/0.1.0/typst.toml:9]: To fix test

```suggestion
exclude = ["example/*","flags.png"]
```

**Comment by reknih** [comment]
Thank you for this package submission. We'd like you to choose a different name for this package since `pride` could be the natural name for packages with these color sets. By taking up this name, the potential for other packages to improve upon this one is reduced. How about a different name like `stonewall` or something else entirely? Excited to see what you come up with!

**Comment by coco33920** [comment]
> Thank you for this package submission. We'd like you to choose a different name for this package since `pride` could be the natural name for packages with these color sets. By taking up this name, the potential for other packages to improve upon this one is reduced. How about a different name like `stonewall` or something else entirely? Excited to see what you come up with!

I really like the name `stonewall` let me modify :)

File: packages/preview/stonewall/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Stonewall: colourful, accurate pride flags colour palette for queer gradients
+You can use the colour palette with _gradients_ for maximum results! 
+For example the code in `example/example.typ` which is
+```typ
+#import "../stonewall.typ": flags
```
Comment (laurmaedje) [inline packages/preview/stonewall/0.1.0/README.md:5]:
```suggestion
#import "@preview/stonewall:0.1.0": flags
```

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**